src/hpc/autoscale/clilib.py

```diff
@@ -1838,6 +1838,19 @@ def add_parser(
             default=default_config,
             required=not bool(default_config),
         ).completer = default_completer  # type: ignore
+
+        if os.getenv("AZURE_HPC_CLI", "0") == "1":
```

Comment:
I really like it, but can we just add a code comment to document it in code? I understand its dev only so perhaps not in user facing docs, but i wouldnt mind if its there also. At the minimum a comment in code would help?